api/v1alpha1/openapi.yaml

```
@@ -3756,8 +3756,26 @@ components:
           description: A list of application content.
           items:
             $ref: "#/components/schemas/ApplicationContent"
+        volumes:
+          type: array
+          description: List of volume providers.
+          items:
+            oneOf:
+              - $ref: "#/components/schemas/ImageVolumeProviderSpec"
       required:
         - inline
+    ImageVolumeProviderSpec:
```

Should this be called OCI Artifact ?
```suggestion
    OciArtifactVolumeProviderSpec:
```

---

Would it _always_ be an OCI artifact or could it also be populated from OCI image? Compare the [OCI Volume Source](https://github.com/kubernetes/enhancements/issues/4639) in K8s.